src/bin/cargo-pio.rs

```diff
@@ -1,14 +1,12 @@
-use std::{
-    env, fs,
-    path::{Path, PathBuf},
-};
+use std::path::{Path, PathBuf};
+use std::{env, fs};
 
 use anyhow::*;
-use log::*;
-
 use clap::{App, AppSettings, Arg, ArgMatches, SubCommand};
```

Comment:
Is there a specific reason you didn't use [`structopt`](https://crates.io/crates/structopt)? I think it would very much reduce the boilerplate code.

Reply:
I don't mind switching to `structop`, but perhaps - a separate subsequent PR.

Reply:
Okay, I'll create an issue for it.